src/app/components/common/dialog/directory-services-monitor/directory-services-monitor.component.html

```diff
@@ -16,11 +16,11 @@ <h4 fxFlex="90%">Directory Services Monitor</h4>
     <ng-container matColumnDef="icon">
       <mat-header-cell *matHeaderCellDef fxFlex="74px"></mat-header-cell>
       <mat-cell *matCellDef="let element" fxFlex="74px">
-        <mat-icon *ngIf="element.state == 'HEALTHY'" class="state-healthy">check_circle</mat-icon>
+        <mat-icon *ngIf="element.state == DirectoryServiceState.Healthy" class="state-healthy">check_circle</mat-icon>
         <mat-icon *ngIf="element.state == 'FAULTED'" class="state-faulted">highlight_off</mat-icon>
```

Comment:
Why don't we do this? Something like use DirectoryServiceState.Faulted instead of 'FAULTED'

Reply:
True, fixed.